### PR TITLE
Add version specification

### PR DESCRIPTION
**Pull request by greg-hellings**
Add support for the user specifying the version of the RPM they want to
install as well as testing for it

Addresses #8

### CODE REVIEW

File: molecule/version/molecule.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    - side_effect
+    - verify
+    - destroy
+verifier:
+  name: testinfra
+  lint:
+    name: flake8
+  additional_files_or_dirs:
+    - ../../shared/tests
+gluster_version: "-3.1.3.0"
```
[the remark this answers is not in the document]
Comment (seandst) [reply molecule/version/molecule.yml]: As an alternative, we could also just have a "scenario_vars" here at the top level by convention, and get it (or something like it) explicitly added to the molecule.yml schema.

[the remark this answers is not in the document]
Comment (greg-hellings) [reply molecule/version/molecule.yml]: Hmm, I haven't seen any evidence of there being such a schema that's tested or validated. I have also added keys to the molecule.yml in at least one other role (gluster_gdeploy for provisioning volumes) and there's never been an issue. Are there active plans to test this schema at some point?

[the remark this answers is not in the document]
Comment (seandst) [reply molecule/version/molecule.yml]: It should be at the top of almost every molecule call, e.g.: 

```
$ molecule cleanup -s basic_subscription                               
--> Validating schema /home/semyers/ve/oasis/roles/rhsm/molecule/sub_unsub/molecule.yml.
Validation completed successfully.
--> Validating schema /home/semyers/ve/oasis/roles/rhsm/molecule/release_set_unset/molecule.yml.
Validation completed successfully.                                             
--> Validating schema /home/semyers/ve/oasis/roles/rhsm/molecule/basic_subscription/molecule.yml.
Validation completed successfully.
```

[the remark this answers is not in the document]
Comment (greg-hellings) [reply molecule/version/molecule.yml]: Hmm. It doesn't seem to be working right. :laughing:

[the remark this answers is not in the document]
Comment (greg-hellings) [reply molecule/version/molecule.yml]: Awesome! Moved that to group_vars.

**Comment by greg-hellings** [comment]
Ping @robled and re-ping @seandst for review.

**Pull request merged**